src/panels/lovelace/hui-view.ts

```diff
@@ -6,7 +6,7 @@ import {
   TemplateResult,
 } from "lit-element";
 
-import "@polymer/paper-fab/paper-fab";
+import "../../components/ha-fab";
```

Comment:
Confused why we import it here as we're also importing it in the `hui-view-editable` ?